### PR TITLE
docs: web simulcast api

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

**Review by triptu — APPROVED** [review]

File: docs/javascript/v2/features/simulcast.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+To turn on simulcast in your room, open the [templates](https://dashboard.100ms.live/templates) in the dashboard and enable it for roles there.
```
Comment (triptu) [inline docs/javascript/v2/features/simulcast.mdx:6]: a screenshot would be helpful

Comment (triptu) [inline docs/javascript/v2/features/simulcast.mdx:6]: let's add a brief description of simulcast

File: docs/javascript/v2/features/simulcast.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+## Responding in the app
```
Comment (triptu) [inline docs/javascript/v2/features/simulcast.mdx:9]: we should rename to section to "Setting Simulcast Layer" and just explain that(i.e. knowing available levels, and action to change).
We can add another section "Layer selection based on Video Element's Dimension" and tell that this is auto handled.

File: docs/javascript/v2/features/simulcast.mdx (hunk 1 above)
Comment (raviteja83) [reply docs/javascript/v2/features/simulcast.mdx:6]: not on prod. deployed to qa recently I think. was waiting for that. will update.

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

**Comment by saikatmitra91** [comment]
lets also mention minimum sdk version requirement for this feature at the top. 

Similar to iOS https://github.com/100mslive/100ms-docs/pull/916/files

File: docs/javascript/v2/features/simulcast.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+## Minimum requirements
```
Comment (arjunattam) [inline docs/javascript/v2/features/simulcast.mdx:6]: currently this section is bleeding into the introduction. can we separate them out?

File: docs/javascript/v2/features/simulcast.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+- SDK versions `@100mslive/hms-video-store@0.8.0`, `@100mslive/react-sdk@0.6.0` or later
+- Simulcast to be enabled from dashboard template
+
+Simulcast is a technique by which a WebRTC client(publisher) encodes the same video stream multipe times in different resolutions and bitrates and sends these to the server(SFU) who then decides who(subscriber) receives which of the published resolutions.
+
+To turn on simulcast in your room, open the [templates](https://dashboard.100ms.live/templates) in the dashboard and enable it for roles there.
```
Comment (arjunattam) [inline docs/javascript/v2/features/simulcast.mdx:13]: i think this feature needs more info, and i've added that in #921. do you think this section is required in this doc - given that it focuses on subscriber-side code?

Comment (raviteja83) [reply docs/javascript/v2/features/simulcast.mdx:13]: If it is already covered we can just link to that.

**Review by arjunattam — APPROVED** [review]

**Pull request merged**